dashboard/app/controllers/api/v1/regional_partners_controller.rb

```diff
@@ -1,4 +1,6 @@
-class Api::V1::RegionalPartnersController < ApplicationController
+class Api::V1::RegionalPartnersController < Api::V1::JsonApiController
```

Comment:
why JsonApiController?

Reply:
Because it's a JSON API :P

That [class](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/controllers/api/v1/json_api_controller.rb) returns forbidden instead of redirecting to login for not logged in users. I don't feel too strongly about this. I could leave it and have it redirect.

Reply:
Ah, makes sense. I don't have a lot of context here so either sounds fine to me.